src/middlewared/middlewared/alert/source/mseries_nvdimm_and_bios.py

```diff
@@ -3,7 +3,7 @@
 from middlewared.alert.base import AlertClass, AlertCategory, AlertLevel, Alert, ThreadedAlertSource
 from middlewared.alert.schedule import IntervalSchedule
 
-WEBUI_SUPPORT_FORM = 'Please contact iXsystems Support using the form in System Settings -> General -> Support'
+WEBUI_SUPPORT_FORM = 'Please contact iXsystems Support using the form in System Settings->General->Support'
```

Comment:
Which form is this?  Do you mean the File Ticket button/form?  If so, maybe spell that out.